src/main/java/com/lorepo/icplayer/client/module/text/DraggableGapWidget.java

```diff
@@ -254,7 +255,16 @@ public void markGapAsWrong() {
 
 	@Override
 	public void setDisabled(boolean disabled) {
-		this.disabled = disabled;
+		this.setDisabled(disabled, false);
+	}
+	
+	public void setDisabled(boolean disabled, boolean internally) {
+		if (!internally) {
```

Comment:
Nie wiem czy tutaj nie byłoby to bardziej czytelne gdyby wydzielić to do osobnej funkcji i wołać w oryginalnym setDisabled i setStyleShowAnswers.
Vide: https://www.martinfowler.com/bliki/FlagArgument.html